cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    duration = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
is it possible to have a movie without duration?

Reply:
The film cannot but have a duration, but we may not know it. 
Or the film has not yet been released. It-s a normal situation if you have site like IMDB